src/libawkward/array/ListOffsetArray.cpp

```diff
@@ -1241,7 +1241,7 @@ namespace awkward {
         Index64 adjustedoffsets(offsets.get()->length());
         Index64 adjustednonzero(nonzero.length());
 
-        struct Error err = awkward_listoffsetarray_getitem_adjust_offsets_64(
+        struct Error err = kernel::listoffsetarray_getitem_adjust_offsets<int64_t>(
```

Comment:
This one was missed: it should be untemplated and named `kernel::listoffsetarray_getitem_adjust_offsets_64`.